### PR TITLE
Fix Bug: Set a React Controller whose view is a RCTRootView as a part…

### PR DESCRIPTION
**Pull request by fishmwei**
Problem representation：   
 
rn-tabbar-bug

## Motivation

To fix a bug I met: Set a React Controller whose view is a RCTRootView as a part of UITabbarController, firstly push to a normal controller, then pop back to the React Controller again. Randomly, the height of RCTContentView is 49 pixels smaller than the height of RCTRootView.

## Test Plan

Currently not sure how to test, but if someone has an idea on how this should be tested I'm happy to hear.

## Release Notes
 [IOS] [BUGFIX]   - Fix the error of the height of the content view when pop back to UITabbarController.

### CODE REVIEW

**Comment by fishmwei** [comment]
@shergin

default

As the log shows, sometime setBounds(7) does not trigger layoutSubview, when setBounds interval too fast . At last,  the height of contentView is 569.  49 pixels smaller than the height of RCTRootView (which backgroundColor is white). In my opinion, it's unnecessary to set available size every time, especially when the size does not equal to its superview. 

Note:　it happened when iOS version < 11 , in iOS 11 , it won't call setFrame twice, may be a bug of iOS.

File: React/Base/RCTRootContentView.m [hunk 1]
```diff
@@ -86,6 +86,14 @@ - (void)updateAvailableSize
     return;
   }
 
+  //unnecessary to update avaliableSize, it's triggered by setBounds: from RCTUIManager
+  if (self.superview) {
+    CGRect superviewFrame = self.superview.bounds;
+    if (!CGRectEqualToRect(superviewFrame, self.frame)) {
```
Comment (shergin) [inline React/Base/RCTRootContentView.m]: Okay, but what is an idea/reasoning behind this changeset?
Comparing `view.frame` with `superview.bounds` makes no any sense, I am affraid.

Comment (fishmwei) [reply React/Base/RCTRootContentView.m]: I always think that the size of content view should be equal to its super view's size. That's why you set content's frame in the function layoutSubviews of rootView ? 
```
- (void)layoutSubviews
{
  [super layoutSubviews];
  _contentView.frame = self.bounds;  // here
  _loadingView.center = (CGPoint){
    CGRectGetMidX(self.bounds),
    CGRectGetMidY(self.bounds)
  };
}
```

if `view.frame`  != `superview.bounds`, it's invalidate

Comment (fishmwei) [reply React/Base/RCTRootContentView.m]: @shergin what's your  opinion?

**Comment by hramos** [comment]
Have you signed the CLA yet?

**Comment by hramos** [comment]
Closing, CLA remains unsigned.

**Pull request closed without merging**